backend/app/Http/Controllers/User/ShowEditUserController.php

```diff
@@ -159,9 +158,10 @@ public function __invoke(
          */
         //学科
         /** @var UUMajor */
-        $uuMajor = UUMajor::where('id', $userInfo->u_u_major_id)->first();
-        $userInfo->u_u_faculty_id = $uuMajor->faculty_id;
-
+        if ($userInfo->u_u_faculty_id) {
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>PHPDoc tag @var does not specify variable name.